MSOfficeUpdates/MSOffice2011UpdateInfoProvider.py

```diff
@@ -43,6 +43,7 @@
 CULTURE_CODE = "0409"
 BASE_URL = "http://www.microsoft.com/mac/autoupdate/%sMSOf14.xml"
 MUNKI_UPDATE_NAME = "Office2011_update"
+URL_DL = "http"
```

Comment:
Oh sorry, I'm mixing up something from a MS issue a few days ago. The xml can be HTTPS, but the links still point to HTTP.